### DISCUSSION TITLE
SmartScript::ProcessAction: SMART_ACTION_TALK:

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#635] [General]
Привет!
У меня появилась ошибка после последнего обновления. Сборку дела в Release. Я чем это связано?

SmartScript::ProcessAction: SMART_ACTION_TALK: EntryOrGuid 28994 SourceType 0 EventType 10 TargetType 7 using non-existent Text id 0 for talker 29494, ignored.
SmartScript::ProcessAction: SMART_ACTION_TALK: EntryOrGuid 29523 SourceType 0 EventType 10 TargetType 7 using non-existent Text id 0 for talker 35500, ignored.
SmartScript::ProcessAction: SMART_ACTION_TALK: EntryOrGuid 28989 SourceType 0 EventType 10 TargetType 7 using non-existent Text id 0 for talker 29493, ignored.
SmartScript::ProcessAction: SMART_ACTION_TALK: EntryOrGuid 28721 SourceType 0 EventType 10 TargetType 7 using non-existent Text id 0 for talker 28701, ignored.
SmartScript::ProcessAction: SMART_ACTION_TALK: EntryOrGuid 28725 SourceType 0 EventType 10 TargetType 7 using non-existent Text id 0 for talker 28703, ignored.
SmartScript::ProcessAction: SMART_ACTION_TALK: EntryOrGuid 28726 SourceType 0 EventType 10 TargetType 7 using non-existent Text id 0 for talker 28699, ignored.
![image](https://github.com/trickerer/Trinity-Bots/assets/99982508/f7b80343-eb45-4c77-8a95-17414733eaa1)

И в самом верху: Error while configuring Appender GM in Logger commands.gm. Appender does not exist
![image](https://github.com/trickerer/Trinity-Bots/assets/99982508/73b218b4-ffba-4aa3-9ef6-a13b24b8ab48)

[Errors.log](https://github.com/trickerer/Trinity-Bots/files/12702634/Errors.log)

**Comment by trickerer** [comment]
Боты тут не при чём
https://github.com/trickerer/Trinity-Bots/issues/631

> И в самом верху: Error while configuring Appender GM in Logger commands.gm. Appender does not exist

Проверь конфиг, возможно строка `Apeender.GM=...` закоментирована
